.gitignore

```diff
@@ -100,7 +122,11 @@ ENV/
 
 # mypy
 .mypy_cache/
+.dmypy.json
+dmypy.json
+
+# Pyre type checker
+.pyre/
 
-# profraw files from LLVM? Unclear exactly what triggers this
-# There are reports this comes from LLVM profiling, but also Xcode 9.
-*profraw
+# IDEA file
+.idea/
```

Comment:
I have always seen your idea files. But not sure if this is something that pycharm does, or if it's your convention.

Reply:
So I added this here to avoid committing idea files.

Reply:
I know what the purpose of the.gitignore file is. I was instead wondering who creates .idea files.